lib/commands.js

```diff
@@ -30,6 +30,7 @@ program.command('build [path]')
   .option('-e, --env [setting]', 'specify a set of override settings to apply')
   .option('-p, --production', 'same as `--env production`')
   .option('-d, --debug [pattern]', 'print verbose debug output to stdout')
+  .option('-J, --parallel [num]', 'parallelize the build')
```

Comment:
Let's remove a CLI flag. That should be specified in configs. Let's keep CLI simple.